cli/cmd/cloud_account.go

```diff
@@ -86,3 +90,200 @@ func cloudAccountsToTable(integrations []api.CloudAccountRaw) [][]string {
 	}
 	return out
 }
+
+func cloudAccountList(_ *cobra.Command, _ []string) error {
+	var (
+		cloudAccounts api.CloudAccountsResponse
+		err           error
+	)
+
+	if cloudAccountType != "" {
+		caType, found := api.FindCloudAccountType(cloudAccountType)
+		if !found {
+			return errors.Errorf("unknown cloud account type '%s'", cloudAccountType)
+		}
+		cloudAccounts, err = cli.LwApi.V2.CloudAccounts.ListByType(caType)
+	} else {
+		cloudAccounts, err = cli.LwApi.V2.CloudAccounts.List()
+	}
+	if err != nil {
+		return errors.Wrap(err, "unable to get cloud accounts")
+	}
+
+	if len(cloudAccounts.Data) == 0 {
+		cli.OutputHuman("No cloud accounts found.\n")
+		return nil
+	}
+
+	if cli.JSONOutput() {
+		return cli.OutputJSON(cloudAccounts.Data)
+	}
```

Comment:
Same here, order matters, switch this if with the one above please. (and any other one I missed)